components/email/test/editorial_signup.coffee

```diff
@@ -63,18 +87,18 @@ describe 'EditorialSignupView', ->
         ARTSY_EDITORIAL_CHANNEL: '123'
         SUBSCRIBED_TO_EDITORIAL: false
       @view.initialize()
-      _.defer ->
-        @view.showEditorialCTA.called.should.not.be.ok()
+      @trigger.callCount.should.equal 0
 
-    it 'old modal is displayed if there arent any auction reminders', ->
+    it 'old modal is displayed if there arent any auction reminders', (done) ->
       @EditorialSignupView.__set__ 'sd',
         ARTICLE: channel_id: '123'
         ARTSY_EDITORIAL_CHANNEL: '123'
         SUBSCRIBED_TO_EDITORIAL: false
       @view.initialize()
       mediator.trigger 'auction-reminders:none'
-      _.defer ->
-        @view.showEditorialCTA.called.should.be.ok()
+      _.defer =>
+        @view.ctaBarView.render.callCount.should.equal 1
+        done()
```

Comment:
Also with a defer we should be using done(). In this case, we want to defer because we should wait for mediator